npc/merchants/refine.txt

```diff
@@ -560,9 +560,20 @@ lhz_in02,282,20,7	script	Fulerr	869,{
 //= .@safe to the max safe refine in refine_db.txt as well.
 //============================================================
 function	script	refinemain	{
-	disable_items;
 	.@npc_name$ = getarg(0);
 	.@features = getarg(1);
+
+	if( getbattleflag( "feature.refineui" ) == 3 ){
```

Comment:
`if( getbattleflag( "feature.refineui" ) == 2 ){`

Reply:
This is fine like this.
1 = enabled in general
2 = enabled in scripts
=> so we want the system to be enabled & enabled in scripts => 3

Reply:
Okay

Reply:
When `2 = enabled in scripts` is configured I still can use `@refineui`